go/vt/vtctl/grpcvtctldserver/server_test.go

```diff
@@ -259,6 +267,185 @@ func TestGetTablet(t *testing.T) {
 	assert.Error(t, err)
 }
 
+func TestGetSchema(t *testing.T) {
+	ctx := context.Background()
+	ts := memorytopo.NewServer("zone1")
+	vtctld := NewVtctldServer(ts)
+
+	validAlias := &topodatapb.TabletAlias{
+		Cell: "zone1",
+		Uid:  100,
+	}
+	testutil.AddTablet(ctx, t, ts, &topodatapb.Tablet{
+		Alias: validAlias,
+	})
+	otherAlias := &topodatapb.TabletAlias{
+		Cell: "zone1",
+		Uid:  101,
+	}
+	testutil.AddTablet(ctx, t, ts, &topodatapb.Tablet{
+		Alias: otherAlias,
+	})
+
+	// we need to run this on each test case or they will pollute each other
+	setupSchema := func() {
+		testutil.TabletManagerClient.Schemas[topoproto.TabletAliasString(validAlias)] = &tabletmanagerdatapb.SchemaDefinition{
+			DatabaseSchema: "CREATE DATABASE vt_testkeyspace",
+			TableDefinitions: []*tabletmanagerdatapb.TableDefinition{
+				{
+					Name: "t1",
+					Schema: `CREATE TABLE t1 (
+	id int(11) not null,
+	PRIMARY KEY (id)
+);`,
+					Type:       "BASE",
+					Columns:    []string{"id"},
+					DataLength: 100,
+					RowCount:   50,
+					Fields: []*querypb.Field{
+						{
+							Name: "id",
+							Type: querypb.Type_INT32,
+						},
+					},
+				},
+			},
+		}
+	}
+
+	tests := []*struct {
```

Comment:
(Bah, GitHub didn't save this question + it's the only blocking question I had.)

🙈 I'm sorry to be the bearer of this question, but did you consider test cases for the `tables` and/or `exclude_tables` parameters?

I'm also wondering if it might be _advantageous_ to mock at > 1 table for these tests.

Reply:
No because those are passed straight through to the tabletmanagerclient, so testing is that is their problem, imo

Reply:
Or I guess, "yes, I considered it; here's why I'm not" to actually answer your question 😂

Reply:
Ohhh I missed that! That makes sense, thanks.